src/core/client/base.js

```diff
@@ -21,29 +21,30 @@ export default function makeClient(routes, createStore) {
   const lang = getLanguage(html.getAttribute('lang'));
   const locale = langToLocale(lang);
   const appName = config.get('appName');
-  // eslint-disable-next-line global-require
-  const jedData = require(`json!../../locale/${locale}/${appName}.json`);
-  const i18n = new Jed(jedData);
-
-  if (initialStateContainer) {
-    try {
-      initialState = JSON.parse(initialStateContainer.textContent);
-    } catch (error) {
-      log.error('Could not load initial redux data');
+  // eslint-disable-next-line global-require, max-len
+  require(`bundle?name=[name]-i18n-[folder]!json!../../locale/${locale}/${appName}.json`)((jedData) => {
```

Comment:
Would be nice to move this into the config but I'm not sure how to do that in a good way at the moment.